pkg/boot/boot.go

```diff
@@ -202,7 +202,11 @@ func buildNamespace(ctx context.Context, tenant string, provider Discovery, clus
 		return nil, err
 	}
 
-	var initCmds = []namespace.Command{namespace.UpdateParameters(cluster.Parameters), namespace.UpdateSlowThreshold(cluster.Parameters)}
+	var initCmds = []namespace.Command{
+		namespace.UpdateSlowLogger(""),
```

Comment:
What's the default slow log path? I think it should be read from bootstrap.yaml or an environment.